tests/unit/test_routes/test_resource_retreival.py

```diff
@@ -144,5 +148,27 @@ def test_filters(module_client, module_db):
     for resource in response.json['data']:
         assert (resource.get('category') == "Back End Dev")
 
-    # TODO: Filter by updated_after
-    # (Need to figure out how to manually set last_updated and created_at)
+    # Filter by updated_after
+    filter_time = datetime.now() + timedelta(days=-1)
+
+    #   Given I update all resources to be last updated 1 week ago.
+    set_resource_last_updated()
+    module_db.session.commit()
```

Comment:
I think we should move the commit into `set_resource_last_updated()` unless there's a reason it needs to be here